### PR TITLE
Continue html and css buildout

### PR DESCRIPTION
**Pull request by knees4bees**
## **_PR TEMPLATE_**
1. Is this a fix or a feature?
* Feature: nearly finished initial HTML and CSS buildout. (Left to do: add drop shadow, add alt text to images, organize CSS -- I made a card for this on the project board.)
* 
2. What is the change?
* Lots of tweaks and small additions in the html and css files.
* 
3. What does it fix?
* N/A
* 
4. Where should the reviewer start?
* Open index.html in the browser and make sure it matches the comp (except for the drop shadow).
* 
5. How should this be tested?
* See #4 above.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,68 +3,61 @@
 <html lang="en" dir="ltr">
   <head>
     <meta charset="utf-8">
-    <!-- insert font info here -->
-    <link href="" rel="stylesheet">
-
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:300">
```
Comment (stevecalla) [inline index.html:6]: Nice change!! Did know how to do that

File: index.html [hunk 2]
```diff
@@ -3,68 +3,61 @@
...
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400">
     <link rel="stylesheet" href="./styles.css">
     <title>Intention Timer</title>
   </head>
   <body>
     <header class="header">
-      <h1 id="h1">INTENTION TIMER</h1>
+      <h1 class="h1">INTENTION TIMER</h1>
```
Comment (stevecalla) [inline index.html]: Why did we create classes for the h1 = “.h1”, h2 = “.h2"? We can use just “h1” et al to target. Maybe we anticipate needing a class later on but if so we should consider a more unique name as “.h1" is repetitive and not unique.

File: index.html [hunk 3]
```diff
@@ -3,68 +3,61 @@
...
-            </p>
-          </form>
-          <form class="time-input">
-            <div>
-              <p>Minutes</p>
-              <p>
-                <label for="time"></label>
-                <input type="text" id="minutes-input" value=""/>
-              </p>
+        <h2 class="h2">New Activity</h2>
```
Comment (stevecalla) [inline index.html]: See above regarding h1, h2, h3 classes.

File: index.html [hunk 4]
```diff
@@ -3,68 +3,61 @@
...
+        <form class="form">
+          <article class="categories">
+            <h3 class="h3">Select a category:</h3>
```
Comment (stevecalla) [inline index.html]: See above regarding h1, h2, h3 classes.

File: index.html [hunk 5]
```diff
@@ -3,68 +3,61 @@
...
+              </div>
+            </section>
+          </article>
+          <article class="accomplish">
+            <h3 class="h3">What would you like to accomplish during this time?</h3>
+            <input type="text" class="accomplish-input" id="accomplishInput" value=""/>
+          </article>
+          <article class="time">
+            <div class="time-container">
+              <h3 class="h3">Minutes</h3>
```
Comment (stevecalla) [inline index.html]: See above regarding h1, h2, h3 classes.

File: index.html [hunk 6]
```diff
@@ -3,68 +3,61 @@
...
-              <p>
-                <label for="time"></label>
-                <input type="text" id="minutes-input" value=""/>
-              </p>
+        <h2 class="h2">New Activity</h2>
+        <form class="form">
+          <article class="categories">
+            <h3 class="h3">Select a category:</h3>
+            <section class="category-box-wrapper">
+              <div class="category-boxes study-box">
```
Comment (stevecalla) [inline index.html]: exercise-box/study-box: did we eliminate/replace the CSS for these? if so we should consider removing these classes. and if so, i’m curious how we got everything to align (no worries).

File: index.html [hunk 7]
```diff
@@ -3,68 +3,61 @@
...
+                <img class="category-image"src="./assets/study.svg">
+                <p class="category-text">Study</p>
+              </div>
+              <div class="category-boxes">
+                <img class="category-image"src="./assets/meditate.svg">
+                <p class="category-text">Meditate</p>
+              </div>
+              <div class="category-boxes exercise-box">
```
Comment (stevecalla) [inline index.html]: exercise-box/study-box: did we eliminate/replace the CSS for these? if so we should consider removing these classes. and if so, i’m curious how we got everything to align (no worries).

File: styles.css [hunk 8]
```diff
@@ -1,122 +1,157 @@
...
-    flex-direction: row;
-    justify-content: space-around;
-    width: 47vw;
+.category-box-wrapper {
+  display: flex;
+  justify-content: space-between;
+  width: 45vw;
 }
 
-.study-box {
```
Comment (stevecalla) [inline styles.css:63]: exercise-box/study-box: did we eliminate/replace the CSS for these? if so we should consider removing these classes. and if so, i’m curious how we got everything to align (no worries).

File: styles.css [hunk 9]
```diff
@@ -1,122 +1,157 @@
...
-    color: red; */
-    margin-right: auto;
+.category-boxes {
+  border: 0.15em solid#FFF;
+  border-radius: 5px;
+  width: 10em;
+  text-align: center;
 }
 
-.exercise-box {
```
Comment (stevecalla) [inline styles.css:69]: exercise-box/study-box: did we eliminate/replace the CSS for these? if so we should consider removing these classes. and if so, i’m curious how we got everything to align (no worries).

File: styles.css [hunk 10]
```diff
@@ -1,122 +1,157 @@
...
 .new-activity {
-    /* border: 0.5em solid black; */
-    display: flex;
-    flex-direction: column;
-    width: 60%;
-    background-color:#2B2733;
+  /* border: 0.5em solid black; */
+  border-right: 0.25em solid #46424D;
+  width: 67%;
+  padding: 0 25px 25px 35px;
```
Comment (stevecalla) [inline styles.css:45]: “px” margins/padding/etc - we should consider changing to em/rem for responsiveness and consistency.

File: styles.css [hunk 11]
```diff
@@ -1,122 +1,157 @@
...
+  text-align: center;
 }
 
-.exercise-box {
-    /* border: 0.5em solid red;
-    color: red; */
-    margin-left: auto;
+.category-image {
+  width: 1.5em;
+  margin-top: 10px;
```
Comment (stevecalla) [inline styles.css:82]: “px” margins/padding/etc - we should consider changing to em/rem for responsiveness and consistency.

File: styles.css [hunk 12]
```diff
@@ -1,122 +1,157 @@
...
 }
 
-.category-boxes {
-    border: 0.1em solid#FFF;
-    width: 10em;
-    text-align: center;
+.category-text {
+  font-size: 1.1rem;
+  font-weight: 300;
+  margin: 0 0 10px 0;
```
Comment (stevecalla) [inline styles.css]: “px” margins/padding/etc - we should consider changing to em/rem for responsiveness and consistency.

File: styles.css [hunk 13]
```diff
@@ -1,122 +1,157 @@
...
 .start-button {
-    /* border: 0.5em solid black; */
-    display: flex;
-    justify-content: flex-end;
-    width: 47vw;
+  border: none;
+  border-radius: 5px;
+  font-size: 1rem;
+  font-weight: 300;
+  padding: 10px 30px;
```
Comment (stevecalla) [inline styles.css]: “px” margins/padding/etc - we should consider changing to em/rem for responsiveness and consistency.

File: styles.css [hunk 14]
```diff
@@ -1,122 +1,157 @@
...
-    width: 39.7%;
-    background-color:#2B2733;
+  /* border: 0.5em solid black; */
+  width: 33%;
+  padding: 0 25px 25px 35px;
+  background-color:#2B2733;
+}
+
+.message-container {
+  padding: 25px 0;
```
Comment (stevecalla) [inline styles.css:146]: “px” margins/padding/etc - we should consider changing to em/rem for responsiveness and consistency.

File: styles.css [hunk 15]
```diff
@@ -1,122 +1,157 @@
...
 }
 
 /* ---------------------PAST ACTIVITY SECTION--------------- */
 .past-activity {
-    /* border: 0.5em solid black; */
-    width: 39.7%;
-    background-color:#2B2733;
+  /* border: 0.5em solid black; */
+  width: 33%;
+  padding: 0 25px 25px 35px;
```
Comment (stevecalla) [inline styles.css:141]: “px” margins/padding/etc - we should consider changing to em/rem for responsiveness and consistency.

File: styles.css [hunk 16]
```diff
@@ -1,122 +1,157 @@
...
+
+.message-container {
+  padding: 25px 0;
+}
+
+.activity-message {
+  font-size: 1rem;
+  font-weight: 300;
+  text-align: center;
+  margin: 10px 0;
```
Comment (stevecalla) [inline styles.css:153]: “px” margins/padding/etc - we should consider changing to em/rem for responsiveness and consistency.

File: index.html [hunk 17]
```diff
@@ -3,68 +3,61 @@
...
-                <label for="time"></label>
-                <input type="text" id="minutes-input" value=""/>
-              </p>
+        <h2 class="h2">New Activity</h2>
+        <form class="form">
+          <article class="categories">
+            <h3 class="h3">Select a category:</h3>
+            <section class="category-box-wrapper">
+              <div class="category-boxes study-box">
+                <img class="category-image"src="./assets/study.svg">
```
Comment (stevecalla) [inline index.html]: Need alt text.

File: index.html [hunk 18]
```diff
@@ -3,68 +3,61 @@
...
+                <p class="category-text">Study</p>
+              </div>
+              <div class="category-boxes">
+                <img class="category-image"src="./assets/meditate.svg">
```
Comment (stevecalla) [inline index.html]: Need alt text.

File: index.html [hunk 19]
```diff
@@ -3,68 +3,61 @@
...
+                <p class="category-text">Meditate</p>
+              </div>
+              <div class="category-boxes exercise-box">
+                <img class="category-image b"src="./assets/exercise.svg">
```
Comment (stevecalla) [inline index.html]: Need alt text.

File: styles.css [hunk 20]
```diff
@@ -1,122 +1,157 @@
...
-    background-color: #46424D;
-    height: 10vh;
+  /* border: 0.5em solid black; */
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  color: #FFF;
+  background-color: #46424D;
+  height: 8.5vh;
```
Comment (stevecalla) [inline styles.css]: Order. Height should be above margin.

File: styles.css (hunk 12 above)
Comment (stevecalla) [inline styles.css]: Order. Margin should be above font details.

File: styles.css [hunk 21]
```diff
@@ -1,122 +1,157 @@
...
+.minutes-input,
+.seconds-input,
+.accomplish-input {
+  border-top: none;
+  border-left: none;
+  border-right: none;
+  border-bottom: 0.15em solid #CBC9CF;
+  width: 100%;
+  background-color: #46424D;
+  color: #FFF;
```
Comment (stevecalla) [inline styles.css]: order. color above background-color to be consistent with prior.

File: styles.css (hunk 13 above)
Comment (stevecalla) [inline styles.css]: order. padding between border and font.

File: styles.css [hunk 22]
```diff
@@ -1,122 +1,157 @@
...
 
-.time-input {
-    /* border: 0.5em solid green; */
-    display: flex;
-    width: 47vw;
-    justify-content:space-between;
+.minutes-input,
+.seconds-input,
+.accomplish-input {
+  border-top: none;
```
Comment (stevecalla) [inline styles.css]: Can we consolidate all border top et al into 1 line?

File: styles.css [hunk 23]
```diff
@@ -1,122 +1,157 @@
...
+}
+
+.h2 {
+  font-size: 1.5rem;
+  font-weight: 300;
+  color: #CBC9CF;
 }
 
 /* ---------------------NEW ACTIVITY SECTION--------------- */
 .new-activity {
```
Comment (stevecalla) [inline styles.css:41]: Box shadow is along these lines. Not sure if this the correct location as the shadow isn't real clear to me in the comp.

box-shadow: 0 19px 38px rgba(0,0,0,0.20), 0 15px 12px rgba(0,0,0,0.

**Review by stevecalla — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (knees4bees) [reply index.html:6]: 🙂

File: index.html (hunk 2 above)
Comment (knees4bees) [reply index.html]: Good point. I did this because I was copying Hannah's code in the Andy comp. Not sure if it's a good idea or not.

Comment (knees4bees) [reply index.html]: Changed class names at @m1073496 's suggestion

File: index.html (hunk 6 above)
Comment (knees4bees) [reply index.html]: Yes, I made a flex container to hold them with `justify-content: space-between` and didn't need these individual classes anymore. I think they can be safely deleted.

File: index.html (hunk 17 above)
Comment (knees4bees) [reply index.html]: added!

File: styles.css (hunk 23 above)
Comment (knees4bees) [reply styles.css:41]: Done! And adjusted while we were on the call.

File: styles.css (hunk 10 above)
Comment (knees4bees) [reply styles.css:45]: Discussed by voice. We'll keep as-is for now.

File: styles.css (hunk 22 above)
Comment (knees4bees) [reply styles.css]: yes! done

**Comment by m1073496** [comment]
Code reviewed and changed together by team. Looks great!

**Pull request merged**